### ISSUE TITLE
Add part-time to contract work

### ISSUE THREAD

**Issue by kalenjordan** [kalenjordan/commercehero-issues#51]
Currently you can specify you want moonlight work, project work and/or full-time work. Can we add part-time contract work?

**Comment by kalenjordan** [comment]
@lfolco might not be a bad idea but what's the use case that would make that different from the other options?

**Comment by lfolco** [comment]
Well, I guess I think of 'project work' as "I build your website and then I'm done". If that's not the case, then I guess project work would cover it? I was thinking of the scenario where you take on a client and maintain their site, rather than build a new one. I could also be over complicating the heck out of this ;)

**Comment by kalenjordan** [comment]
Yeah...might be something there - in practice I'm not sure I see clients looking for this level of detail - if anything hours of availability per $time_period might handle this?
